src/main/java/net/minecraftforge/client/event/EntityViewRenderEvent.java

```diff
@@ -3,10 +3,8 @@
 import net.minecraft.block.Block;
 import net.minecraft.client.renderer.EntityRenderer;
 import net.minecraft.entity.Entity;
-import net.minecraft.entity.EntityLivingBase;
 import net.minecraftforge.fml.common.eventhandler.Cancelable;
 import net.minecraftforge.fml.common.eventhandler.Event;
-import net.minecraftforge.fml.common.eventhandler.Event.Result;
```

Comment:
I didn't originally intend to do this, and this is the only import change I have done.